lib/common/multithread.c

```diff
@@ -87,11 +87,16 @@ static void init_async(h2o_multithread_queue_t *queue, h2o_loop_t *loop)
 {
     int fds[2];
 
+#ifndef _WIN32
     if (cloexec_pipe(fds) != 0) {
         perror("pipe");
         abort();
     }
     fcntl(fds[1], F_SETFL, O_NONBLOCK);
+#else
+    u_long nonblock = 1;
```

Comment:
> libuv seemed like the easiest one as it already supports windows and mingw-w64.
> Your thoughts ?

The H2O standalone server only supports the embedded event loop (evloop).  Libuv binding exists for users of libh2o (since it is a popular event loop that is used, with rich documentation and other protocol bindings that application developers can use).

I agree that porting the libuv-side would be faster, but if your intention is to make the H2O standalone server runnable on Windows, you would need to port evloop.